runtime/drivers/clickhouse/clickhouse.go

```diff
@@ -105,9 +108,17 @@ type configProperties struct {
 	// EmbedPort is the port to run Clickhouse locally (0 is random port).
 	EmbedPort int `mapstructure:"embed_port"`
 	// DataDir is the path to directory where db files will be created.
-	DataDir        string `mapstructure:"data_dir"`
-	TempDir        string `mapstructure:"temp_dir"`
-	CanScaleToZero bool   `mapstructure:"can_scale_to_zero"`
+	DataDir string `mapstructure:"data_dir"`
+	TempDir string `mapstructure:"temp_dir"`
+
+	// CanScaleToZero is used to determine if the OLAP instance can be scaled to zero.
+	// In absence of configs required to make requests to the cloud API or in case of API errors service status will be set to this flag.
+	CanScaleToZero bool `mapstructure:"can_scale_to_zero"`
+	// CloudAPIKeyID, CloudAPIKeySecret, OrganizationID, ServiceID are used to make requests to the cloud API
+	APIKeyID       string `mapstructure:"api_key_id"`
+	APIKeySecret   string `mapstructure:"api_key_secret"`
+	OrganizationID string `mapstructure:"organization_id"`
+	ServiceID      string `mapstructure:"service_id"`
```

Comment:
These fields should probably be prefixed with `cloud_` to clarify that they are CH Cloud specific.